cmd/up/up.go

```diff
@@ -158,6 +158,9 @@ func Up() *cobra.Command {
 			os.Setenv(model.OktetoNameEnvVar, oktetoManifest.Name)
 
 			if len(oktetoManifest.Dev) == 0 {
+				if oktetoManifest.Type == model.StackType {
+					return fmt.Errorf("couldn't find any dev environment from your docker compose %q. Only services with local volumes are transformed into devs: https://www.okteto.com/docs/reference/compose/#volumes-string-optional", oktetoManifest.Name)
```

Comment:
Let's make this more user centric, since this error us more likely to hit new users. Maybe something like:

"Your docker compose file is not currently supported: Okteto requires a "volume to be defined. See %url for more information."